src/fiatExchanges/normalizeQuotes.ts

```diff
@@ -0,0 +1,194 @@
+import { FiatAccountType, FiatAccountTypeQuoteData } from '@fiatconnect/fiatconnect-types'
+import { FiatExchangeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { FiatConnectQuote } from 'src/fiatconnect'
+import {
+  SUPPORTED_FIAT_ACCOUNT_SCHEMAS,
+  SUPPORTED_FIAT_ACCOUNT_TYPES,
+} from 'src/fiatconnect/FiatDetailsScreen'
+import {
+  CICOFlow,
+  FetchProvidersOutput,
+  isSimplexQuote,
+  PaymentMethod,
+  SimplexQuote,
+} from 'src/fiatExchanges/utils'
+import i18n from 'src/i18n'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { navigateToURI } from 'src/utils/linking'
+
+export interface NormalizedQuote {
+  quote: QuoteInfo
+  provider: ProviderInfo
+}
+
+export interface QuoteInfo {
+  paymentMethod: PaymentMethod
+  fee: number | null
+  kycInfo: string | null
+  timeEstimation: string
+  onPress: () => void
+}
+
+export interface ProviderInfo {
+  name: string
+  logo: string
+  id: string
+}
+
+const strings = {
+  oneHour: i18n.t('selectProviderScreen.oneHour'),
+  numDays: i18n.t('selectProviderScreen.numDays'),
+  idRequired: i18n.t('selectProviderScreen.idRequired'),
+}
+
+function getSettlementEstimation(lower: string | undefined, upper: string | undefined) {
+  // TODO: Dynamically generate time estimation strings
+  //
+  return strings.numDays
+}
+
```

Comment:
would be great to get a docstring here, even just 1 sentence to explain what "normalize" means